src/Components/HistoryCard/HistoryCard.jsx

```diff
@@ -0,0 +1,28 @@
+import styles from "./HistoryCard.module.css";
+const HistoryCard = (props) => {
```

Comment:
you can destruct props here to avoid using props everywhere like ((title, thumbnail ,etc}) ....and the just use {title} instead of props.title